os/board/sidk_s5jt200/src/s5jt200_tash.c

```diff
@@ -196,7 +196,7 @@ static void sidk_s5jt200_configure_partitions(void)
 		} else
 #endif
 #if defined(CONFIG_MTD_SMART) && defined(CONFIG_FS_SMARTFS)
-		if (!strncmp(types, "smartfs,", 8)) {
+		if (!strncmp(types, "jSmartFs,", 9)) {
```

Comment:
Are we sure that the mount command's working has been updated for all boards?
What about rtk8721csm
rtk8720e
stm32....

I don't see any changes pertaining to those boards here.

Basically, in os/board/ -> all the possible devices would have their bringups/tash files that need tobe modified so that they can mount with the name jSmartFs instead of smartfs

Reply:
(1)
TASH>>df -h
Filesystem    Size      Used  Available Mounted on
jSmartFs     3912K      461K      3451K /mnt
TASH>>ls -l
/:
 dr--r--r--       0 dev/
 drw-rw-rw-       0 mnt/
(2) 
TASH>>echo "Hello" > f2.txt
TASH>>ls -l
/mnt:
 -rw-rw-rw-       6 f2.txt
(3)
TASH>>pwd
         /mnt
(4)
TASH>>cat f2.txt
Hello
(5)
TASH>>mount -t jSmartFs /dev/smart0p7 /mnt
smartfs_mount:      SMARTFS:
smartfs_mount:      Sector size:     1024
smartfs_mount:      Bytes/sector     1018
smartfs_mount:      Num sectors:     3912
smartfs_mount:      Free sectors:    3450
smartfs_mount:      Max filename:    32
smartfs_mount:      RootDirSector:   3
smartfs_sector_recovery: ###############################
smartfs_sector_recovery: #      FS Recovery Report     #
smartfs_sector_recovery: ###############################
smartfs_sector_recovery: Total of active sectors : 453
smartfs_sector_recovery: Recovered Isolated Sectors : 0
smartfs_sector_recovery: Cleaned Entries : 0

(6) 
TASH>>umount /mnt
TASH>>ls
/:
dev/